CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## 4.12.0 (Unreleased)
 
+FEATURES:
+
+* **New Data Source:** `cloudflare_zone_cache_reserve` ([#2642](https://github.com/cloudflare/terraform-provider-cloudflare/issues/2642))
```

Comment:
this file is automatically updated on merge so you don't need to commit this.

Reply:
Got it. I wasn't sure how much of a changelog would be required. Will fix.

Reply:
Done. Thank you!